src/widgets/dragAndDrop.ts

```diff
@@ -48,30 +54,32 @@ function makeDropTarget (ele, droppedURIHandler, droppedFileHandler) {
     }
   }
 
-  var dropListener = function (e) {
+  const dropListener = function (this: any, e: DragEvent) {
```

Comment:
I think I have more comments about this in the draganddrop branch, but in order to use this. you have to pass in this:any, there were a couple more ideas, but this seem to be the best option.